test/math/ExpDecayMath.ts

```diff
@@ -13,6 +12,7 @@ describe('ExpDecayMath', () => {
     let expDecayMath: TestExpDecayMath;
     const ONE = new Decimal(1);
     const TWO = new Decimal(2);
+    const MAX = new Decimal(128);
```

Comment:
Not aligned with latest changes in the ExpDecay PR

Reply:
merged with the latest changes